root/static/scripts/edit/components/FormRowTextListSimple.js

```diff
@@ -0,0 +1,177 @@
+/*
```

Comment:
This file is identical to the previous version of FormRowTextList.js, minus the updated name.

Unfortunately, duplicating it was much easier than adding a bunch of conditionals to determine the shape of the repeatable subfield.

Reply:
A comment should probably be added to the file itself about it being deprecated and not to be used for new cases, at least, I'd say? Possibly also a bit more of an explanation about the difference with the non-`Simple` version.

Reply:
Can you check the current comment and see if anything needs adding?